src/map/utils/charutils.cpp

```diff
@@ -1780,7 +1780,6 @@ namespace charutils
                     PChar->look.sub            = 0;
                     PChar->m_Weapons[SLOT_SUB] = itemutils::GetUnarmedItem(); // << equips "nothing" in the sub slot to prevent multi attack exploit
                     PChar->health.tp           = 0;
-                    PChar->StatusEffectContainer->DelStatusEffect(EFFECT_AFTERMATH);
```

Comment:
This line should stay. When TP gets reset aftermath gets cleared unless you have video showing otherwise.